### PR TITLE
feature/actors

### PR DESCRIPTION
**Pull request by tora-kozic**
Support the actors V1 APIs in the SDK and CLI.

I very likely should've broke this into 2 tickets

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,27 @@
...
+
+- Support for [Actors APIs](https://developer.code42.com/api/#tag/Actors), including:
+  - An `actors.v1` client to the SDK with the following methods:
+    - `client.actors.v1.get_page()` - to query a single page of actors.
+    - `client.actors.v1.iter_all()` - to lazily iterate through all pages of actors.
+    - `client.actors.v1.get_actor_by_id()` - to retrieve details of a single actor by ID.
+    - `client.actors.v1.get_actor_by_name()` - to retrieve details of a single actor by name.
+    - `client.actors.v1.get_family_by_member_id()` - to retrieve details of an actor family by a member's ID.
+    - `client.actors.v1.get_family_by_member_name()` - to retrieve details of an actor family by a member's name.
+    - `client.actors.v1.create_adoption()` - to create child-parent adoptions between actors.
```
Comment (tora-kozic) [inline CHANGELOG.md:24]: these are called `create_adoption()` and `remove_adoption` but they could also be called `add_parent()` and `remove_parent()` I think - lmk thoughts

Comment (johnsonjben) [reply CHANGELOG.md:24]: do know if either of those terms are things our customers are already familiar with?

**Review by johnsonjben — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,27 @@
  how a consumer would use the library or CLI tool (e.g. adding unit tests, updating documentation, etc) are not captured
  here.
 
+## 1.3.0 (Unreleased)
+
+### Added TODO
```
Comment (jdullum) [inline CHANGELOG.md:14]: TODO

**Review by jdullum — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (tora-kozic) [reply CHANGELOG.md:24]: My understanding is there's been all sorts of product back & forth on how much to communicate around families and parent-child adoption. I had them add some description on the concepts to the dev portal but outside of that I don't think there's much.

Comment (schuyler-c42) [reply CHANGELOG.md:24]: Let's go with your initial instinct and keep `create_adoption()` and `remove_adoption`! 

Reason: that's the language we have in the API Reference documentation, and I think we keep it consistent!

**Review by jdullum — APPROVED** [review]

**Pull request merged**